wasm/src/account/private_key.rs

```diff
@@ -100,6 +100,14 @@ impl PrivateKey {
         Signature::sign(self, message)
     }
 
+    /// Sign an instance of a valid Aleo data type or record.
+    ///
+    /// @param {String} message The string representation of the Aleo datatype or record to sign.
+    /// @returns {Signature} Signature of the message.
+    pub fn sign_value(&self, message: &str) -> Result<Signature, String> {
```

Comment:
Also ensure to `js_name` any `snake_case` method into `camelCase`.

```suggestion
    #[wasm_bindgen(js_name = signValue)]
    pub fn sign_value(&self, message: &str) -> Result<Signature, String> {
```

Reply:
[3ce2b9b](https://github.com/ProvableHQ/sdk/pull/1099/commits/3ce2b9b7586510bee61cb410be3f87377a45437d)